src/_pyawaitable/coro.c

```diff
@@ -11,11 +12,17 @@ awaitable_send_with_arg(PyObject *self, PyObject *value)
     if (aw->aw_gen == NULL)
     {
         PyObject *gen = awaitable_next(self);
-        if (gen == NULL)
+        if (PyAwaitable_UNLIKELY(gen == NULL))
         {
             return NULL;
         }
 
+        if (PyAwaitable_UNLIKELY(value != Py_None)) {
+            PyErr_SetNone(PyExc_RuntimeError,
```

Comment:
Should this be ``PyErr_SetString`` as this compile errors saying there are is more than 1 arg passed into it and that it accepts only ``PyObject *`` (3.14 source clone of cpython used to check this).